@file:Suppress("unused", "UNUSED_PARAMETER", "PackageDirectoryMismatch")

package androidx.activity.result

abstract class ActivityResultLauncher<I> {
    abstract fun launch(input: I)
    open fun unregister() {}
}
